frontend/src/component/common/Limit/Limit.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { screen } from '@testing-library/react';
+import { render } from 'utils/testRenderer';
+import { Limit } from './Limit';
+
+test('Render approaching limit variant', () => {
+    render(
+        <Limit
+            name='strategies in this environment'
+            shortName='strategies'
+            limit={10}
+            currentValue={8}
+        />,
+    );
+
+    screen.getByText(
+        'You are nearing the limit for strategies in this environment',
+    );
+    screen.getByText('80%');
+    screen.getByText('Limit: 10');
+});
+
+test('Render reached limit variant', () => {
+    render(
+        <Limit
+            name='strategies in this environment'
+            shortName='strategies'
+            limit={10}
+            currentValue={10}
+        />,
+    );
+
+    screen.getByText(
+        'You reached the limit for strategies in this environment',
```

Comment:
This is a suggestion for the component, but I'll comment here because it was the first place I saw it:

```suggestion
        'You have reached the limit for strategies in this environment',
```

We should add the "have" in at the start.